src/settings.ini

```diff
@@ -48,18 +48,24 @@ swapLeftCtrlLeftAltAndLeftWin=0
 # Unterstützung der Ebenen 5 und 6 (experimentell)
 supportLevels5and6=1
 
-# CapsLock key acts as Escape when hit alone
-# CapsLock-Taste sendet Escape, wenn sie alleine angeschlagen wird
+# CapsLock key acts as Escape when hit alone. Also see modTapTimeout.
+# CapsLock-Taste sendet Escape, wenn sie alleine angeschlagen wird. Siehe auch modTapTimeout.
 capsLockAsEscape=0
 
-# the right level 3 modifier acts as Return when hit alone (experimental)
-# der rechte Ebene3-Modifier sendet Return, wenn er alleine angeschlagen wird (experimentell)
+# The right level 3 modifier acts as Return when hit alone. Also see modTapTimeout. (experimental)
+# Der rechte Ebene3-Modifier sendet Return, wenn er alleine angeschlagen wird. Siehe auch modTapTimeout. (experimentell)
 mod3RAsReturn=0
 
-# the left level 4 modifier acts as Tab when hit alone (experimental)
-# der linke Ebene4-Modifier sendet Tab, wenn er alleine angeschlagen wird (experimentell)
+# The left level 4 modifier acts as Tab when hit alone. Also see modTapTimeout. (experimental)
+# Der linke Ebene4-Modifier sendet Tab, wenn er alleine angeschlagen wird. Siehe auch modTapTimeout (experimentell)
 mod4LAsTab=0
 
+# If capsLockAsEscape, mod3RAsReturn and/or mod4LAsTab are enabled, the modifier key will only act as the regular
+# key when it was not down longer than this timeout (miliseconds). 0 means infinity. (experimental)
+# Falls capsLockAsEscape, mod3AsReturn und/oder mod4LAsTab aktiviert sind, sendet die Modifier-Taste die normale
+# Taste nur, wenn sie nicht länger als dieser Timeout (Milisekunden) gehalten wurde. 0 bedeutet unendlich. (experimentell)
+modTapTimeout=0
```

Comment:
I suppose this setting name let's the user assume it's valid for `ModTap` keys (as defined below). It should be clarified that this is not the case. Or even better: This timeout should be used for these `ModTap` keys, too.

...or we just keep the name for now, and I implement it later for the other `ModTap` keys...

Reply:
I could give it some long, descriptive name of course. I'd say, this is your call though 🤷